src/defu.ts

```diff
@@ -28,7 +24,9 @@ function _defu<T extends defuObj> (baseObj: T | any, defaults: T | any): T {
       continue
     }
 
-    if (isObject(val) && isObject(obj[key])) {
+    if (Array.isArray(val) && Array.isArray(obj[key])) {
+      obj[key] = obj[key].concat(val)
+    } else if (isObject(val) && isObject(obj[key])) {
```

Comment:
[`isObject`](https://github.com/nuxt-contrib/defu/blob/b1da67e8ec49aa94a632381cc0285e569d6edaba/src/defu.ts#L2) is still doing additional `Array.isArray` check

Reply:
Updated